bindings/pyroot_experimental/PyROOT/python/ROOT/pythonization/_tobject.py

```diff
@@ -0,0 +1,79 @@
+# Author: Enric Tejedor CERN  02/2019
+
+################################################################################
+# Copyright (C) 1995-2019, Rene Brun and Fons Rademakers.                      #
+# All rights reserved.                                                         #
+#                                                                              #
+# For the licensing terms see $ROOTSYS/LICENSE.                                #
+# For the list of contributors see $ROOTSYS/README/CREDITS.                    #
+################################################################################
+
+from ROOT import pythonization
+import cppyy
+
+# Searching
+
+def _contains(self, o):
+    # Relies on TObject::FindObject
+    # Parameters:
+    # - self: object where to search
+    # - o: object to be searched in self
+    # Returns:
+    # - True if self contains o
+    return bool(self.FindObject(o))
+
+# Comparison operators
+
+def _eq(self, o):
+    if isinstance(o, cppyy.gbl.TObject):
+        return self.IsEqual(o)
+    else:
+        return False
+
+def _ne(self, o):
+    if isinstance(o, cppyy.gbl.TObject):
+        return not self.IsEqual(o)
+    else:
+        return True
+
+def _lt(self, o):
+    if isinstance(o, cppyy.gbl.TObject):
+        return self.Compare(o) == -1
+    else:
+        return NotImplemented
```

Comment:
`NotImplemented` is kind of unspecific. Wouldn't it be better to run in an exception like `raise Exception("Comparison of {} and {} is not valid.".format(self, o))`?

Reply:
Or ` TypeError`?

Reply:
Ah ok, I see: https://docs.python.org/3/library/constants.html#NotImplemented

Reply:
Ok, it works exactly as you've done it. `NotImplemented` gives Python the possibility to try another operator before throwing an error. The name is weird though ...

Reply:
Yes, seems the way to go for binary special methods. I will try the other operand upon return of `NotImplemented`.